### PR TITLE
Add 3.12 to pylint testing and use .x

### PR DESCRIPTION
**Pull request by Soapy7261**
i dunno how else to word it
dont merge until you merge toothy's pr (otherwise all the other jobs will cancel once 3.12 fails)

### CODE REVIEW

**Comment by ToothyDev** [comment]
Do we really need it all the way back to 3.8

**Comment by JoshuaSlui** [comment]
Yes.

**Comment by Soapy7261** [comment]
3.8 is the earliest version that works (according to pylint) since 3.7 does not work since py-cord does not support it

**Comment by Soapy7261** [comment]
image

**Comment by JoshuaSlui** [comment]
Please remove 3.8, 3.9 and 3.10 from pylint. I don't want these tested.
Focus on staying up to date and not backwards compatibility for 5 versions long.

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8.16", "3.9.16", "3.10.11", "3.11.3"]
+        python-version: ["3.8.x", "3.9.x", "3.10.x", "3.11.x", "3.12.x"]
```
Comment (JoshuaSlui) [inline .github/workflows/pylint.yml]:
```suggestion
        python-version: ["3.11.x", "3.12.x"]
```

Comment (Soapy7261) [reply .github/workflows/pylint.yml]: Alright

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**